lib/release_notes_bot/persists.ex

```diff
@@ -157,4 +220,12 @@ defmodule ReleaseNotesBot.Persists do
   defp replace_spaces_with_plus_signs(string), do: String.replace(string, " ", "+")
 
   defp drop_question_mark(string), do: String.replace(string, "?", "")
+
+  defp update_confluence_page() do
+    {:error, "Not implemented"}
+    # curl -u admin:admin -X PUT -H 'Content-Type: application/json' -d '{"id":"3604482","type":"page",
+    # "title":"new page","space":{"key":"TST"},"body":{"storage":{"value":
+    # "<p>This is the updated text for the new page</p>","representation":"storage"}},
+    # "version":{"number":2}}' http://localhost:8080/confluence/rest/api/content/3604482
```

Comment:
If this is an artifact for testing code, could you identify it as such. Otherwise it could be mistaken for dead code.

Reply:
This will be purged - it was helpful during development but is no longer necessary